src/util/main.rs

```diff
@@ -62,6 +62,14 @@ fn main() {
             return
         },
         _ => {
+	    match opts.syncword {
```

Comment:
`if let` is a good pattern if you only need one of the match options

```suggestion
	    if let Some(syncword) = opts.syncword {
```